sru-scripts/manual/oracle-sru

```diff
@@ -78,6 +93,6 @@ while ! ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- cloud-init status --wait --long; do
 done
 echo 'After reboot'
 ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- "grep 'Update datasource' /var/log/cloud-init.log"
+oci compute instance terminate --force --instance-id="${INSTANCE_ID}"
```

Comment:
Looks good, however if the script fails at some point we won't reach this `oci compute instance terminate` line, and it's very likely to forget to manually cleanup the stale instance. What do do think about adding a cleanup function like:
```
function cleanup {
    echo "Cleaning up instance ${INSTANCE_ID}"
    oci compute instance terminate --force --instance-id="${INSTANCE_ID}"
}
```
at the beginning of the script, and a
```
trap cleanup EXIT
```
line right after the `oci compute instance launch` line?